packages/create/src/Generator.js

```diff
@@ -1,11 +1,23 @@
 import path from 'path';
+import commandLineArgs from 'command-line-args';
+
 import { copyTemplates, copyTemplate, copyTemplateJsonInto, installNpm } from './core.js';
+import { parseCliOptions } from './helpers.js';
+
+const optionDefinitions = [
+  { name: 'tag-name', type: String, defaultValue: '' },
+  { name: 'npm-install', type: String, defaultValue: 'true' }, // set to emptry string later
+  { name: 'scaffold', type: String, defaultValue: '' },
+];
+
+export const cliOptions = commandLineArgs(optionDefinitions, { partial: true });
 
 class Generator {
   constructor() {
     this._destinationPath = process.cwd();
     this.templateData = {};
-    this._installNpm = true;
+    this.wantsNpmInstall = '';
```

Comment:
shouldn't that be `true` instead of an empty string? 🤔 

also this option is set to a different value in `generators/app` pls update there as well 🤗

Reply:
the command-line-args module doesnt seem to support having a default undefined value for booleans, so this is the only way to have 3 states/possibilities (default = `''`, force yes = `'true'`, force no = `'false'`)